theme/snippets/add-to-cart-container.liquid

```diff
@@ -0,0 +1,47 @@
+<!-- Assign label and disabled state to add to cart button -->
+{% capture button_label %}{{"snippets.add_to_cart_container.add_to_cart_btn.text" | t }}{% endcapture %}
+{% assign button_disabled = false %}
+{% if product.available %}
+  {% if product_in_cart %}
+    {% capture button_label %}{{"snippets.add_to_cart_container.added_to_cart_btn.text" | t }}{% endcapture %}
+    {% assign button_disabled = true %}
+  {% endif %}
+{% else %}
+  {% case product_type %}
+    {% when "Course" %}
+    {% when "Event" %}
+      {% capture button_label %}{{"snippets.add_to_cart_container.registration_closed_btn.text" | t }}{% endcapture %}
+      {% assign button_disabled = true %}
+    {% else %}
+      {% capture button_label %}{{"snippets.add_to_cart_container.sold_out_btn.text" | t }}{% endcapture %}
+      {% assign button_disabled = true %}
+  {% endcase %}
+{% endif %}
+
+<div class="AddToCartContainer">
+  <form data-add-to-cart-form data-product-type={{product_type}} action="/cart/add" method="post" enctype="multipart/form-data" id="AddToCartForm">
+    <div class="AddToCartContainer__quantity-select-wrapper overflow-hidden relative mt_625{% if product.variants.size <= 1 %} mb1_75 {% endif %}">
+      <select class="sans-xs uppercase radius-xs border-black w100 pl1_125 py_5" name="quantity" id="Quantity" data-quantity-selector>
+        <option value="" selected="selected">{{"snippets.add_to_cart_container.quantity_spinner.label" | t }}</option>
+        {% for i in (1...10) limit: product.selected_or_first_available_variant.inventory_quantity %}
```

Comment:
I am trying to only show amounts in the quantity select dropdown, based on the quantity available. If 3 available, show numbers 1 - 3.

Reply:
Maybe there's a better way to do this?

Reply:
We decided to keep this hardcoded to 10 and I added a to-do to add some kind of notification if the user tries to add an amount > product inventory